modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -177,7 +177,7 @@ The *Custom metrics* tab displays only if you have a Titanium subscription and s
 
 The *Application Network* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. 
 
-The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. Additionally, it shows only direct callers and callees for the selected app over the most recent seven days. The apps must be deployed, and the connections are based on the selected time range.
+The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. The selected app must be started. Additionally, the chart shows only direct callers and callees for the selected app. The connections are based on the selected time range with a max of the most recent seven days.
```

Comment:
I suggest a slight rewording: 

The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app, which must be started. The chart shows only direct callers and callees for the selected app.

Reply:
```suggestion
The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app, which must be started. The chart shows only direct callers and callees for the selected app. The connections are based on the selected time range, with a max of the most recent seven days.
```